notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/ArbeidsgiverNotifikasjonKlientUtils.kt

```diff
@@ -34,13 +34,14 @@ fun ArbeidsgiverNotifikasjonKlient.opprettSak(
         }
     }
 
-fun ArbeidsgiverNotifikasjonKlient.ferdigstillSak(sakId: String) {
+fun ArbeidsgiverNotifikasjonKlient.ferdigstillSak(sakId: String, linkUrl: String, forespoerselId: UUID) {
```

Comment:
Det var vel slik jeg spesifiserte, men kunne vurdert å slå sammen de to parametrene link og forespørsel til en linkUrl, så får vi to argumenter inn heller enn tre. Men ingen biggie, dere kan endre eller beholde as-is.

Reply:
Bra forslag, det! 😄 I fix 👍